### DISCUSSION TITLE
Update image tag without applying in the kubernetes cluster

### DISCUSSION THREAD

**Discussion by jerzykiller** [fluxcd/flux2#2874] [Q&A] [answered]
Hi. I bootstraped a brand new fluxv2 installation with `--components-extra=image-reflector-controller,image-automation-controller`.  Everything works great, it scans for new tag, updates the tag and apply it to the cluster. Is it possible to remove or stop applying new image to the cluster? I would like to use Argocd for that.

To sum up, I would like to skip the last step
```
Production CI/CD workflow

DEV: push a bug fix to the app repository
DEV: bump the patch version and release e.g. v1.0.1
CI: build and push a container image tagged as registry.domain/org/app:v1.0.1
CD: pull the latest image metadata from the app registry (Flux image scanning)
CD: update the image tag in the app manifest to v1.0.1 (Flux cluster to Git reconciliation)
CD: deploy v1.0.1 to production clusters (Flux Git to cluster reconciliation)
```

Thanks for your help

**Comment by stefanprodan** [comment, marked as answer]
Flux write the update to Git, then it syncs it on the cluster, so this is no possible. Flux be design does not change the cluster state unless the change is recorded in Git.
